app-misc/jigit/jigit-1.20.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="A tool to ease the distribution of very large files"
+DESCRIPTION="Jigsaw Download, or jigdo, is a tool designed to ease the \
+distribution of very large files over the internet such as CD or DVD \
+images. Its aim is to make downloading these images as easy as a click \
+on a direct download link in a browser, while avoiding all the problems \
+that server administrators could have with hosting large files."
+HOMEPAGE="http://www.einval.com/~steve/software/JTE/"
+COPYRIGHT="2014 Steve McIntyre"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://www.einval.com/~steve/software/JTE/download/jigit_$portVersion.orig.tar.gz"
+CHECKSUM_SHA256="459285bd743645381912301578316f2bfafb99ce1c10140a01af2b6cb011f193"
+PATCHES="jigit-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	jigit = $portVersion
+	cmd:jigit = $portVersion
+	cmd:jigdump = $portVersion
+	cmd:jigsum = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libbz2
+	lib:libz
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libbz2
+	devel:libz
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
```

Comment:
Probably:
```diff
-	makefile_engine
```
makefile_engine is only needed for programs written for BeOS or Haiku.